pkg/apis/extensions/types.go

```diff
@@ -413,6 +421,10 @@ const (
 	// Replace the old daemons by new ones using rolling update i.e replace them on each node one after the other.
 	RollingUpdateDaemonSetStrategyType DaemonSetUpdateStrategyType = "RollingUpdate"
 
+	// Replace the old daemons by new ones using rolling update i.e replace them on each node one
+	// after the other, creating the new pod and then killing the old one.
```

Comment:
This is not different from using a Rolling deployment with a RWO volume. Worth calling out though.